lib/task/migrate/arUpgradeSqlTask.class.php

```diff
@@ -340,6 +329,49 @@ protected function updateDatabaseVersion($version)
     QubitPdo::modify($sql, array($version, 'version'));
   }
 
+  /**
+   * Run new style migrations in a directory
+   *
+   * @param string $migrationsDirectory  Directory with migrations in it
+   * @param int $previousMilestone  Previous milestone
+   * @param int $currentMilestone  Current milestone
+   */
+  private function runMigrationsInDirectory($migrationsDirectory, $previousMilestone, $currentMilestone)
+  {
+    foreach (sfFinder::type('file')
+      ->maxdepth(0)
+      ->sort_by_name()
+      ->name('arMigration*.class.php')
+      ->in($migrationsDirectory) as $filename)
+    {
+      // Initialize migration class
+      $className = preg_replace('/.*(arMigration\d+).*/', '$1', $filename);
+      $class = new $className;
+
+      // This upgrade should have been applied already
+      if ($class::VERSION <= $version)
+      {
+        // Unless the user is moving from 1.x to 2.x
+        if (2 == $class::MIN_MILESTONE && 1 == $previousMilestone && 2 == $currentMilestone)
+        {
+          // Run migration but don't bump dbversion
+          if (true !== $class->up($this->configuration)) throw new sfException('Failed to apply upgrade '.get_class($class));
```

Comment:
I think this is a good opportunity to make this "if () throw ..." statement (and the one below) follow our coding standard and add the parantheses and newlines, e.g. 

```
if (true !== $class->up($this->configuration)) 
{
  throw new sfException('Failed to apply upgrade '.get_class($class));
}
```